### ISSUE TITLE
Not loading added plugins

### ISSUE THREAD

**Issue by mltytskr** [MscrmTools/XrmToolBox#226]
I downloaded the latest XrmToolBox (1.2015.7.31) and tried to add AndyPopkin.CRMBulkWorkflowTool.dll and the DLaB.LocalToGlobalOptionSet.Plugin dlls.  If XrmToolBox is open, it knows the files were added and asks if I want to refresh the plugin list, however, the plugins don't show up in the list.  They are also not listed in settings.  I tried to grab an older version that I had saved on my network drive (1.2015.7.6) and that is doing the same thing it used to work before I rebuilt my computer with Windows 10.  Is there some way to force load plugins or clear everything out?  I have tried to delete the folder and redownload both the tool box and plugins.

**Comment by MscrmTools** [comment]
I guess those plugins have not been migrated for the latest version of XrmToolBox.
You can check if the plugins are supported for the latest version here : https://github.com/MscrmTools/XrmToolBox/wiki/Available-plugins-for-XrmToolBox (look for the green mark).
If it is supported, please download the latest version of this plugin to be sure it works with latest version of XrmToolBox. 
If it is not yet supported, then you can try to contact the developer of this plugin to ask when it will be.

**Comment by mltytskr** [comment]
Thank you, that was part of the problem, I also had to unblock the files that were downloaded from the internet.  After that I was getting an error about the background and found out those were migrated.  I was able to find an updated version of the Local to Global option sets on Codeplex, but not for the bulk workflow tool.  I have contacted the developer to see if he would update.  I was able to grab the old XrmToolBox off of Codeplex in the mean time.  Thanks again.
